VS_Project/Pokerapp/Pokerapp/delete-ajax.cshtml

```diff
@@ -0,0 +1,19 @@
+﻿@using WebMatrix.Data
```

Comment:
Wederom lijkt het er hier op dat je geen html gebruikt, dus waarom staat dit niet in een class?